Source/CocoaMQTT.swift

```diff
@@ -29,33 +29,33 @@ import CocoaAsyncSocket
 /**
  * Conn Ack
  */
-@objc public enum CocoaMQTTConnAck: UInt8, CustomStringConvertible {
-    case accept  = 0
-    case unacceptableProtocolVersion
-    case identifierRejected
-    case serverUnavailable
-    case badUsernameOrPassword
-    case notAuthorized
-    case reserved
-    
-    public var description: String {
-        switch self {
-            case .accept:                       return "accept"
-            case .unacceptableProtocolVersion:  return "unacceptableProtocolVersion"
-            case .identifierRejected:           return "identifierRejected"
-            case .serverUnavailable:            return "serverUnavailable"
-            case .badUsernameOrPassword:        return "badUsernameOrPassword"
-            case .notAuthorized:                return "notAuthorized"
-            case .reserved:                     return "reserved"
-        }
-    }
-}
+//@objc public enum CocoaMQTTConnAck: UInt8, CustomStringConvertible {
+//    case accept  = 0
+//    case unacceptableProtocolVersion
+//    case identifierRejected
+//    case serverUnavailable
+//    case badUsernameOrPassword
+//    case notAuthorized
+//    case reserved
+//    
+//    public var description: String {
+//        switch self {
+//            case .accept:                       return "accept"
+//            case .unacceptableProtocolVersion:  return "unacceptableProtocolVersion"
+//            case .identifierRejected:           return "identifierRejected"
+//            case .serverUnavailable:            return "serverUnavailable"
+//            case .badUsernameOrPassword:        return "badUsernameOrPassword"
+//            case .notAuthorized:                return "notAuthorized"
+//            case .reserved:                     return "reserved"
+//        }
+//    }
+//}
```

Comment:
Do we need to keep this commented-out code?

Reply:
Hi, the pr is not finalized yet. It will not be merged until further progress is made. At the same time, there is some conversation about MQTT 5.0 behind this pr is going in. Please be looking forward to our new updates ~!